cupy_backends/cuda/api/runtime.pxd

```diff
@@ -469,117 +530,6 @@ cpdef enum:
-    cudaDevAttrMaxTexture2DGatherWidth = 45
-    cudaDevAttrMaxTexture2DGatherHeight = 46
-    cudaDevAttrMaxTexture3DWidthAlt = 47
-    cudaDevAttrMaxTexture3DHeightAlt = 48
-    cudaDevAttrMaxTexture3DDepthAlt = 49
-    cudaDevAttrPciDomainId = 50
-    cudaDevAttrTexturePitchAlignment = 51
-    cudaDevAttrMaxTextureCubemapWidth = 52
-    cudaDevAttrMaxTextureCubemapLayeredWidth = 53
-    cudaDevAttrMaxTextureCubemapLayeredLayers = 54
-    cudaDevAttrMaxSurface1DWidth = 55
-    cudaDevAttrMaxSurface2DWidth = 56
-    cudaDevAttrMaxSurface2DHeight = 57
-    cudaDevAttrMaxSurface3DWidth = 58
-    cudaDevAttrMaxSurface3DHeight = 59
-    cudaDevAttrMaxSurface3DDepth = 60
-    cudaDevAttrMaxSurface1DLayeredWidth = 61
-    cudaDevAttrMaxSurface1DLayeredLayers = 62
-    cudaDevAttrMaxSurface2DLayeredWidth = 63
-    cudaDevAttrMaxSurface2DLayeredHeight = 64
-    cudaDevAttrMaxSurface2DLayeredLayers = 65
-    cudaDevAttrMaxSurfaceCubemapWidth = 66
-    cudaDevAttrMaxSurfaceCubemapLayeredWidth = 67
-    cudaDevAttrMaxSurfaceCubemapLayeredLayers = 68
-    cudaDevAttrMaxTexture1DLinearWidth = 69
-    cudaDevAttrMaxTexture2DLinearWidth = 70
-    cudaDevAttrMaxTexture2DLinearHeight = 71
-    cudaDevAttrMaxTexture2DLinearPitch = 72
-    cudaDevAttrMaxTexture2DMipmappedWidth = 73
-    cudaDevAttrMaxTexture2DMipmappedHeight = 74
-    # Use header version
-    # cudaDevAttrComputeCapabilityMajor = 75
-    # cudaDevAttrComputeCapabilityMinor = 76
-    cudaDevAttrMaxTexture1DMipmappedWidth = 77
-    cudaDevAttrStreamPrioritiesSupported = 78
-    cudaDevAttrGlobalL1CacheSupported = 79
-    cudaDevAttrLocalL1CacheSupported = 80
-    cudaDevAttrMaxSharedMemoryPerMultiprocessor = 81
-    cudaDevAttrMaxRegistersPerMultiprocessor = 82
-    cudaDevAttrManagedMemory = 83
-    cudaDevAttrIsMultiGpuBoard = 84
-    cudaDevAttrMultiGpuBoardGroupID = 85
-    cudaDevAttrHostNativeAtomicSupported = 86
-    cudaDevAttrSingleToDoublePrecisionPerfRatio = 87
-    cudaDevAttrPageableMemoryAccess = 88
-    cudaDevAttrConcurrentManagedAccess = 89
-    cudaDevAttrComputePreemptionSupported = 90
-    cudaDevAttrCanUseHostPointerForRegisteredMem = 91
-    cudaDevAttrReserved92 = 92
-    cudaDevAttrReserved93 = 93
-    cudaDevAttrReserved94 = 94
-    cudaDevAttrCooperativeLaunch = 95
-    cudaDevAttrCooperativeMultiDeviceLaunch = 96
-    cudaDevAttrMaxSharedMemoryPerBlockOptin = 97
-    cudaDevAttrCanFlushRemoteWrites = 98
-    cudaDevAttrHostRegisterSupported = 99
-    cudaDevAttrPageableMemoryAccessUsesHostPageTables = 100
-    cudaDevAttrDirectManagedMemAccessFromHost = 101
-    # added since CUDA 11.0
-    cudaDevAttrMaxBlocksPerMultiprocessor = 106
-    cudaDevAttrReservedSharedMemoryPerBlock = 111
-    # added since CUDA 11.1
-    cudaDevAttrSparseCudaArraySupported = 112
-    cudaDevAttrHostRegisterReadOnlySupported = 113
-    # added since CUDA 11.2
-    cudaDevAttrMaxTimelineSemaphoreInteropSupported = 114
-    cudaDevAttrMemoryPoolsSupported = 115
```

Comment:
@leofang These enum lost in this PR, causing test failures in CUDA 11.2

Reply:
> E   AttributeError: module 'cupy_backends.cuda.api.runtime' has no attribute 'cudaDevAttrMemoryPoolsSupported'

Reply:
shoot, let me send a pr @kmaehashi